src/relayserver/RelayServer.ts

```diff
@@ -232,19 +102,19 @@ export class RelayServer extends EventEmitter {
 
     // Check the relayWorker (todo: once migrated to multiple relays, check if exists)
     const workerIndex = 0
-    if (req.relayWorker.toLowerCase() !== this.getWorkerAddress(workerIndex).toLowerCase()) {
+    if (req.relayWorker.toLowerCase() !== this.workerAddress.toLowerCase()) {
       throw new Error(
         `Wrong worker address: ${req.relayWorker}\n`)
     }
 
     // if trusted paymaster, we trust it to handle fees
-    if (!this.trustedPaymasters.includes(req.paymaster.toLowerCase())) {
+    if (!this.config.trustedPaymasters.map(it => it.toLowerCase()).includes(req.paymaster.toLowerCase())) {
```

Comment:
Can you do the lowercasing in the constructor?

Reply:
No, as I don't touch the 'config' in the constructor, only assign it. 
Having to lowercase addresses in constructor would be awkward.